src/tools/environment.js

```diff
@@ -27,6 +29,12 @@ const cleanEnvironment = () => {
     delete process.env.AWS_SESSION_TOKEN;
     delete process.env.AWS_SECRET_ACCESS_KEY;
   }
+
+  // Lambda may reuse container, which leaves leftovers in process.env. Let's clean
+  // up Zapier specific stuff here.
+  for (const name of ENV_VARS_TO_CLEAN) {
```

Comment:
It might be cleaner (and more functional) to:

`ENV_VARS_TO_CLEAN.map((key) => { delete process.env[key]; });`

Reply:
Fixed 7edea2f . Changed it to `forEach` as we don't need it to return an array like `map` does.